### PR TITLE
Act Header

### PR DESCRIPTION
**Pull request by mikeselander**
This is the page header, based off of the Act page designs. This excludes the entire-page skewed, stepped background that provides enough darkness for the light text. I feel as though that should be in a separate PR.

Based off of #31 - so that PR must be reviewed & merged first.

### CODE REVIEW

**Comment by kadamwhite** [comment]
Some things I noticed when testing locally:

Call To Action button height is off on Mobile size, too thin vs design
image

Rollover on links looks like it's the default, I assume we have some other standard styling to use for links that would look better:
image

Bullets get included on tag link rollover, the `::after` is getting treated as part of the link both stylistically and functionally:
image

File: src/scss/common/_covers.scss [hunk 1]
```diff
@@ -46,112 +46,81 @@
...
+  box-shadow: 0 0 10px rgba(0,0,0,.35);
+  background-size: cover;
+  background-position: center;
+  &:before {
+    content: "";
+    opacity: 0;
+    transition: opacity 100ms linear;
+  }
+  &:hover {
+    @include background-before-opacity(#385924);
```
Comment (kadamwhite) [inline src/scss/common/_covers.scss]: Ought to be a color variable

Comment (mikeselander) [reply src/scss/common/_covers.scss]: 🤦‍♂️ I should've known of all the people too

**Comment by mikeselander** [comment]
Thanks for the review @kadamwhite - I think I'd like to address the button issue overall since we have so many variations of buttons and this looks wrong on mobile on all of them. I'll make an issue for tracking it.

**Pull request merged**